lib/Listener/NewMessageClassificationListener.php

```diff
@@ -67,11 +67,16 @@ public function handle(Event $event): void {
 			}
 		}
 
+		// if the message is already flagged as important, we won't classify it again.
+		$messages = array_filter($event->getMessages(), function ($message) {
+			return ($message->getFlagImportant() === false);
```

Comment:
```suggestion
			return !$message->getFlagImportant();
```

Reply:
in theory it could be also `null`. hence I'm not against the explicit `false` comparison actually